test/Sentry.Tests/SentryClientTests.verify.cs

```diff
@@ -16,4 +16,18 @@ public Task CaptureEvent_BeforeEventThrows_ErrorToEventBreadcrumb()
 
         return Verify(@event.Breadcrumbs);
     }
+
+    [Fact]
+    public Task CaptureTransaction_BeforeSendTransactionThrows_ErrorToEventBreadcrumb()
```

Comment:
This is not working on my end. Is there any additional configuration required for Verify that I'm missing?

Reply:
There shouldn't be.  We did have an unrelated failure due to verify, but I've merged that in to this branch already.

Reply:
PEBCAK. Fixing it 🙂